pom.xml

```diff
@@ -218,10 +218,11 @@
     <it.failsafe.argLine></it.failsafe.argLine>
 
     <!-- Sonar & JaCoCo (related profile: sonar)-->
-    <sonar.host.url>https://sonar.nuxeo.org/</sonar.host.url>
+    <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+    <sonar.organization>nuxeo</sonar.organization>
     <sonar.dynamicAnalysis>reuseReports</sonar.dynamicAnalysis>
     <it.jacoco.destFile>${java.io.tmpdir}/jacoco-it.dump</it.jacoco.destFile>
-    <sonar.jacoco.itReportPath>${it.jacoco.destFile}</sonar.jacoco.itReportPath>
+    <sonar.jacoco.reportPaths>${it.jacoco.destFile}</sonar.jacoco.reportPaths>
```

Comment:
Since we're now on Maven 3, couldn't this be replaced with `${session.executionRootDirectory}` or https://github.com/jdcasey/directory-maven-plugin#directory-of-goal ?